docs/standard/net-standard.md

```diff
@@ -21,7 +21,7 @@ The various .NET implementations target specific versions of .NET Standard. Each
 
 ## .NET implementation support
 
-The following table lists the minimum platform versions that support each .NET Standard version.
+The following table lists the **minimum** platform versions that support each .NET Standard version. That means that later versions listed for each platform are also supported. For example, you can use any version of .NET Standard with .NET Core 2.2.
```

Comment:
Reasoning behind my suggestions:
- The "...later versions listed for each platform..." phrase can be read as "...later versions *that* are listed for each platform...", which is misleading.
- Two sentences use the "support" verb in different directions. In the first, *platform* supports *.NET Standard* (active voice); in the second, *platform* is supported (passive voice). I've rephrased to the active voice.
- The last sentence is not future-proof: .NET Core 2.2 is not going to support .NET Standard 2.1

```suggestion
The following table lists the **minimum** platform versions that support each .NET Standard version. That means that later versions of the listed platform also support the corresponding .NET Standard version. For example, .NET Core 2.2 supports .NET Standard 2.0 and earlier.
```

Reply:
I like the suggestions. Once we add the 2.1 column though and 3.0 releases, I was thinking of changing the example to 3.0. So maybe we say "For example, .NET Core 3.0 supports .NET Standard 1.0 and later versions."?

Reply:
> For example, .NET Core 3.0 supports .NET Standard 1.0 and later versions.

As far as I understand, readers are curious about the platform versions that are *not* in the table. .NET Core 3.0 will be in the table, because it will be the minimum version that would support .NET Standard 2.1; so, no questions about it, but might be a lot about .NET Core 2.1 and 2.2. Thus, I think the example with the *not listed* platform might be more helpful, especially when .NET Core 3.0 is released: that would be *not listed, not latest* platform.

By the way, ".NET Core 3.0 supports .NET Standard 1.0 and later versions" is also not future-proof, as I don't expect .NET Core 3.0 to support .NET Standard 2.2 (or any next version after .NET Standard 2.1)

Reply:
That's a good point about showing a version that won't be on the table. Let's go with your version then!